docs/UserGuide.md

```diff
@@ -71,6 +71,22 @@ Examples:
 * `add c n/Apple p/98765432 e/applehr@example.com d/Top tech company t/tech t/interested`:
   ![add company with all information added](images/addCompany.png)
 
+### Adding an internship to a company contact: `add i`
+
+Adds an internship of interest to the corresponding company.
+
+Format: `add i n/ROLE_NAME d/DESCRIPTION [s/SCHEDULED_INTERVIEW_TIME]`
+* Internship's `ROLE_NAME` and `DESCRIPTION` are compulsory fields.
+* `SCHEDULED_INTERVIEW_TIME` is optional and can only be used once.
+
+Restrictions:
+* `ROLE_NAME` and `DESCRIPTION` must be alphanumeric.
```

Comment:
I think no need alphanumeric? Otherwise your role name and description can't have spaces in it.

Reply:
Both can be any non-empty string and should not be blank

for internships